ui/src/app/components/help-sidebar.tsx

```diff
@@ -373,6 +381,7 @@ export const HelpSidebar = fp.flow(
       bodyPadding?: string;
       renderBody: () => JSX.Element;
       showFooter: boolean;
+      showNavigation?: boolean;
```

Comment:
is this intended to do more than show the back button?  if not, maybe `showBackNavigation`?

I'm not sure that it's the right choice to have a back button, however.  We're assuming here that "back" means the Apps Panel, but I can also navigate to the Cromwell Config Panel by clicking the Cromwell icon.  For this flow, "back" is not meaningful, or it means "close"

Reply:
I don't have strong preferences about the back button. I was thinking that the navigation was something that we might be building out over time. I saw it in the mockup:
https://www.sketch.com/s/64a4f7f4-b8ca-41c7-91f9-b7d1de397a2b/a/WKJKGxw

If it doesn't make sense, I am good with taking it out.